Source/Kesmai.Server/Game/Entities/Mobiles/Creatures/Undead/Banshee.cs

```diff
@@ -6,25 +6,41 @@ namespace Kesmai.Server.Game
 {
 	public partial class Banshee : CreatureEntity, IUndead
 	{
+
+		public override CreatureImmunity Immunity { get; set; } = CreatureImmunity.Piercing;
+		public override CreatureImmunity Immunity { get; set; } = CreatureImmunity.Slashing;
+		public override CreatureImmunity Immunity { get; set; } = CreatureImmunity.Bashing;
+		public override CreatureImmunity Immunity { get; set; } = CreatureImmunity.Projectile;
+		public override CreatureImmunity Immunity { get; set; } = CreatureImmunity.Poison;
+		public override CreatureWeakness Weakness { get; set; } = CreatureWeakness.Silver;
 		/// <summary>
 		/// Initializes a new instance of the <see cref="Banshee"/> class.
 		/// </summary>
 		public Banshee()
 		{
 			Name = "banshee";
 			Body = 12;
-
 			Alignment = Alignment.Chaotic;
+			DeathResistance = 100;
```

Comment:
where are these values coming from?  Ice and death have always hurt them, you can hide from them and stun them.  Is there some method behind these values?  I see you quoting dnd references, but this isn't dnd and trying to make it that probably won't work without making it an entirely different game.

Reply:
Other collaborators have determined that base classes of monsters should be D&D manual accurate, changing the balance of almost every zone with undead, so I'm helping out and making undead manual accurate.

I noticed that Death resistance will only give 50% damage reduction, not immunity. I can fix that.